samples/snippets/src/main/java/AuthenticateImplicitWithAdc.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.google.cloud.compute.v1.Instance;
+import com.google.cloud.compute.v1.InstancesClient;
+import java.io.IOException;
+
+public class AuthenticateImplicitWithAdc {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(Developer):
+    //  1. Before running this sample, authenticate using ADC as mentioned in:
```

Comment:
// 1. Before running this sample, set up ADC as described in https://cloud.google.com/docs/authentication/external/set-up-adc
// 2. (same)
// 3. Make sure that the user account or service account that you are using has the required permissions. For this sample, you must have "compute.instances.list".

Reply:
Thanks. Modified.